bspump/mqtt_metrics/__init__.py

```diff
@@ -136,11 +145,19 @@ def on_message(self, client, userdata, message):
                 L.warning(f"Payload sent to {topic} is not a valid JSON. Payload: {payload}")
                 return
             
-            method = payload.get("method")
-            if method is not None and method == "get":
+            topology_count = payload.get("topology_count")
```

Comment:
Here we end up with a thing were we have two interfaces, one for events and the other for topologies and their field names differ only in that one is `event_count` and the other is `topology_count`. Since we are already in the context of a subscribe maybe this should just be `count` so that the same field name can be used. It can lead to easier code deduplication on the front end.

Reply:
Yeah that makes sense